### ISSUE TITLE
Formatting Bug

### ISSUE THREAD

**Issue by lioneaglesolutions** [Stillat/vscode-antlers-language-server#14]
When formatting with Antlers Language for Statamic, it breaks one of my components
```blade
{{#
    @name Button attributes
    @desc A single button component. Faux is used when a button is displayed inside a link (for example in link blocks).
#}}

{{ if label }}
<{{ as or 'a' }} {{ attribute }} class="btn
        {{if type === 'white'}}
            {{ 'bg-white text-primary ' }}
        {{else }}
            {{ outlined ? 'bg-gradient-to-tr from-white to-neutral-50 border-primary  text-primary border-2' : 'bg-gradient-to-tr from-primary to-primary-300 text-white' }}
        {{ /if }}
        {{ class }} disabled:opacity-50 disabled:cursor-default" 
        {{ unless faux }} {{ partial:snippets/button_attributes }} {{ /unless }}>
    {{ label }}
</{{ as or 'a' }}>
{{ /if }}
```

Becomes

```blade
{{ /unless }}{{#
    @name Button attributes
    @desc A single button component. Faux is used when a button is displayed inside a link (for example in link blocks).
#}}

{{ if label }}
    <{{ as or 'a' }}
    {{ attribute }} class="btn
    {{ if type === 'white' }}
        {{ 'bg-white text-primary ' }}
    {{ else }}
        {{ outlined ? 'bg-gradient-to-tr from-white to-neutral-50 border-primary  text-primary border-2' : 'bg-gradient-to-tr from-primary to-primary-300 text-white' }}
    {{ /if }}
    {{ class }} disabled:opacity-50 disabled:cursor-default"
    {{ unless faux }}
        {{ partial:snippets/button_attributes }} >
        {{ label }}
        </{{ as or 'a' }}>
{{ /if }}
```

With the `{{ /unless}}` moved to the top of the file...

**Comment by JohnathonKoster** [comment]
Thanks for the bug report! I've been able to reproduce this issue, and will be working on a fix throughout the week.

Just to set some timeline expectations: formatter related issues do tend to take longer to fix.

**Comment by lioneaglesolutions** [comment]
Thanks @JohnathonKoster - appreciate the quick response!

**Comment by JohnathonKoster** [comment]
A fix for `/unless` being placed at the beginning of the file has been released, and is now available in the VS Code Marketplace as `v1.1.3`.

Let me know how this works for you!

**Comment by lioneaglesolutions** [comment]
@JohnathonKoster it seems to have fixed the `{{ /unless }}` issue however now the line breaking seems to have changed...
```
---
variants:
    default: "btn bg-gradient-to-tr"
    primary: "from-primary to-primary-300 text-white"
    outlined: "from-white to-neutral-50 border-primary text-primary border-2"
    white: "bg-white text-primary"
    disabled: "opacity-50 cursor-default"
---


{{ if label }}
    {{ if  disabled && attr_title }} 
    <div class="has-tooltip"> 
        <span class="tooltip">{{ attr_title }} </span>
    {{ /if }}
    <{{ as or 'a' }} {{ attribute }} class="{{ view:variants:default }} {{ array_get:view:variants :key="variant:value" }} {{ disabled ? view:variants:disabled : '' }}
            {{ class }} disabled:opacity-50 disabled:cursor-default" 
            {{ unless faux }} {{ partial:snippets/button_attributes }} {{ /unless }}>
        {{ label }}
    </{{ as or 'a' }}>
    {{ if disabled && attr_title }} 
    </div> 
    {{ /if }}
{{ /if }}
```

Becomes
```
---
variants:
    default: "btn bg-gradient-to-tr"
    primary: "from-primary to-primary-300 text-white"
    outlined: "from-white to-neutral-50 border-primary text-primary border-2"
    white: "bg-white text-primary"
    disabled: "opacity-50 cursor-default"
---

{{ if label }}
    {{ if  disabled && attr_title }}
        <div class="has-tooltip">
            <span class="tooltip">
                {{ attr_title }}
            </span>
    {{ /if }}
    <{{ as or 'a' }}
    {{ attribute }} class="
    {{ view:variants:default }}
    {{ array_get:view:variants :key="variant:value" }}
    {{ disabled ? view:variants:disabled : '' }}
    {{ class }} disabled:opacity-50 disabled:cursor-default"
    {{ unless faux }}
        {{ partial:snippets/button_attributes }}
    {{ /unless }}>
    {{ label }}
    </{{ as or 'a' }}>
    {{ if disabled && attr_title }}
        </div>
    {{ /if }}
{{ /if }}
```

I don't think it was doing this before.
